src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ExtensibleChoiceParameterDefinition.java

```diff
@@ -445,9 +445,12 @@ public ParameterValue getDefaultParameterValue()
     {
         ChoiceListProvider p = getEnabledChoiceListProvider();
         String defaultChoice = (p != null)?p.getDefaultChoice():null;
-        if(defaultChoice != null)
-        {
-            return createValue(defaultChoice);
+        if (defaultChoice != null) {
+            try {
+                return createValue(defaultChoice);
+            } catch (IllegalArgumentException x) { // JENKINS-39647
+                return null;

```

Comment:
Just passing-through (and returning the top value) looks a better behavior to me.